packages/ui/src/components/InviteFriendsToTlonButton.tsx

```diff
@@ -67,6 +67,13 @@ export function InviteFriendsToTlonButton({ group }: { group?: db.Group }) {
     }
   }, [group, branchDomain, branchKey, toggle, status, isGroupAdmin, describe]);
 
+  if (
+    group?.privacy === 'private' ||
+    (group?.privacy === 'secret' && !isGroupAdmin)
```

Comment:
```suggestion
    (group?.privacy === 'private' ||
    group?.privacy === 'secret') && !isGroupAdmin
```

Seems like we want this? Otherwise if it's private the group admin won't be able to invite either.